backend/routers/psOne.js

```diff
@@ -29,9 +29,17 @@ router.get('/api/1', async (req, res) => {
     queries.location = { $regex: new RegExp(req.query.location, 'i') };
   }
 
+  if (req.query.industryDomain) {
+    queries.industryDomain = { $regex: new RegExp(req.query.industryDomain, 'i') };
+  }
+
+  if (req.query.branch) {
+    queries.branches = { $in: [new RegExp(req.query.branch, 'i')] };
```

Comment:
I had recommended using the $in operator to get rid of the regex
```javascript
if (req.query.branch) {
    const branches = req.req.query.tags.split(' ');
    queries.branches = { $in: branches };
}
This will allow filtering multiple branches at once. when they are provided as a comma separated list.
```

Reply:
Okay. I'll replace the code with this. Sorry, I had not used regex before. Thanks for helping out.